blastai/engine.py

```diff
@@ -379,7 +380,8 @@ async def get_metrics(self):
             },
             "concurrent_browsers": len([t for t in tasks.values() if t.executor and t.executor.browser_session]),
             "memory_usage_gb": round(memory_gb, 2),
-            "total_cost": round(total_cost, 2)
+            "total_cost": round(total_cost, 2),
+            "total_token_usage": total_token_usage.format_detailed()
```

Comment:
Let me know the preferred display option here! (can also hide token usage entirely)